src/bun.js/ipc.zig

```diff
@@ -230,6 +238,7 @@ const json = struct {
             }
 
             const deserialized = str.toJSByParseJSON(globalThis);
+            // if (deserialized == .zero) return error.JSError;
```

Comment:
this line probably shouldn't be commented out because toJSByParseJSON potentially throws a syntax error and it should not crash in that case.